x-pack/test/detection_engine_api_integration/security_and_spaces/tests/update_rules.ts

```diff
@@ -150,6 +150,9 @@ export default ({ getService }: FtrProviderContext) => {
         await createLegacyRuleAction(supertest, createRuleBody.id, connector.body.id);
```

Comment:
Below here, I add some notes of where I think this test is ensuring a different set of behaviors. The `getSimpleRuleUpdate('rule-1');` is attaching an empty actions array indicating to the system they want to remove the actions altogether.